vale/.vale.ini

```diff
@@ -0,0 +1,8 @@
+StylesPath = /etc/vale/styles
+MinAlertLevel = suggestion
+
+Packages = https://github.com/jdbaldry/Hugo/releases/download/v0.2.0-rc.1/Hugo.zip
```

Comment:
I'm curious if we could add the Google package here, and separate what are the Google rules and the Grafana rules? (I'm also not sure if we have a lot of different rules from the Google package?)

But also as a nit, don't mean to stop this from getting the PR merged. :shipit:

Reply:
Done in [da2adc5](https://github.com/grafana/writers-toolkit/pull/287/commits/da2adc547f2d66ed9b65c73f17d4bef5d77b0abd)

Reply:
Nice! :shipit: